src/uwtools/drivers/ww3.py

```diff
@@ -0,0 +1,82 @@
+"""
+A library driver for ww3.
```

Comment:
```suggestion
An assets driver for ww3.
```
To agree with our chosen nomenclature...